src/WebJobs.Extensions.EasyTables/EasyTableAttribute.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace WebJobs.Extensions.EasyTables
+{
+    /// <summary>
+    /// Attribute used to binds a parameter to an EventTable type.
```

Comment:
Done.

---
In reply to: [54793317](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/45#discussion_r54793317) [](ancestors = 54793317)